File: core/src/com/mygdx/game/entity/ConfirmResult.java
package com.mygdx.game.entity;

public class ConfirmResult extends BaseResult {
    private boolean isOk = false;

    public ConfirmResult(int type) {
        super(type);
    }

    public boolean isOk() {
        return isOk;
    }

    public boolean isNotOK() {
        return !isOk;
    }

    public ConfirmResult setOk(boolean ok) {
        isOk = ok;
        return this;
    }
}

File: core/src/main/kotlin/com/rich/diy/game/entity/ConfirmResult.kt
package com.rich.diy.game.entity

class ConfirmResult(type: Int) : BaseResult(type) {
    var isOk = false
        private set
    val isNotOK: Boolean
        get() = !isOk

    fun setOk(ok: Boolean): ConfirmResult {
        isOk = ok
        return this
    }
}